TCL4 Data Management/utm-tcl4-dmp-con1.yaml

```diff
@@ -1,12 +1,11 @@
-swagger: '2.0'
 info:
-  version: "v1"
+  version: "v1.0"
   title: UTM TCL4 DMP CON1 Specification
   description: Data model for CON1 submission. 
```

Comment:
Add more desc.  If we are on this path of a special model per Scenario, that should be clarified here.  Also be explicit that we expect one instance of this per operation within that scenario.